README.md

```diff
@@ -226,14 +211,14 @@ enter the container using `docker exec`.
 
     ```
     docker exec -it ccm_web /bin/bash
-    ````
+    ```
 
 2. Run one of a handful of test commands defined in `package.json` under `scripts`.
 
     ```
     # To run the unit tests and generate a coverage report
     npm run test:cov
-
+    
```

Comment:
Nit-pick: not sure why the indentation here is necessary.

Reply:
The Markdown editor I used added that.  It's more of an authority on the syntax than I.  I suspect the indentation on blank line is used to designate that it is one continuous indented block, not two indented blocks.  Seems redundant, since the block is delimited by triple-backticks.